tiproxy/tiproxy-configuration.md

```diff
@@ -229,6 +238,7 @@ TLS object fields:
 + `ca`: specifies the CA
 + `cert`: specifies the certificate
 + `key`: specifies the private key
++ `cert-allowed-cn`: when other components connect to TiProxy with TLS, TiProxy can prevent illegal access by verifying the `Common Name` in the caller's certificate. This item specifies a list of `Common Name` of legal callers. After setting this item, this TLS object must enable TLS, otherwise the item will not take effect. For more information on verifying component caller's identity, see [verify component caller's identity](/enable-tls-between-components.md#verify-component-callers-identity).
```

Comment:
```suggestion
+ `cert-allowed-cn`: when other components connect to TiProxy with TLS, TiProxy can prevent invalid access by verifying the `Common Name` in the caller's certificate. This item specifies a list of `Common Name` of legal callers. After setting this item, this TLS object must enable TLS; otherwise the item will not take effect. For more information on verifying component caller's identity, see [verify component caller's identity](/enable-tls-between-components.md#verify-component-callers-identity).
```